src/app/games/abalone/abalone.component.ts

```diff
@@ -214,9 +214,11 @@ export class AbaloneComponent extends HexagonalGameComponent<AbaloneRules,
     private getArrowPath(single: boolean, direction: HexaDirection): { start: Coord, end: Coord, pointed: Coord } {
         let start: Coord = this.selecteds[0];
         let end: Coord = this.selecteds[this.selecteds.length - 1];
+        // If end and start are different, then single === false
         if (single === false &&
             start.getDirectionToward(end).get().equals(direction.getOpposite()))
-        {
+        { // If start towars end give us the opposite direction fo the direction provided in args:
+            // Then we draw the arrow starting from end, so that the longer arrow is always drawned
```

Comment:
manque un espace supplémentaire après le {, ou plus idéalement un \n
towars -> towards
fo -> for
drawned -> drawn
tu l'as fait exprès ? :D

Reply:
Bah si 
1. l'utilisateur sélectionne K en premier, puis L (pour indiquer qu'il veut bouger le groupe KL)
2. le groupe KL peut aller dans la direction anti-alphabétique (de L vers K) 
Alors:
tu ne veux pas une flèche qui fait K->J, tu veux une flèche qui fait L->J, la plus longue !

Aucune flèche n'as été inversé dans le faisage de ce code !

Reply:
Ok je comprends ! Mais donc : 

  - pourquoi passer un booléen en argument qui vaut `this.selectedLengths === 1` si on a déjà `this.selectedLengths` ? J'enlèverai le if de `showPossibleDirection` et calculerait `onlyOnePieceIsSelected` dans les fonctions qui en ont besoin plutôt que de passer ça en argument tout le long, ça permettrait de mieux savoir d'où il vient.
  - y'a pas moyen de formuler ça à un niveau plus haut comme tu viens de le faire ? "Si on a sélectionné (K, L) et qu'on veut une flèche jusque J, on dessine la flèche la plus longue entre (K, J), soit (L, J)"